src/cascadia/TerminalSettingsModel/JsonUtils.h

```diff
@@ -630,7 +630,8 @@ namespace Microsoft::Terminal::Settings::Model::JsonUtils
                 for (const auto& pair : TBase::mappings)
                 {
                     if (pair.second != AllClear &&
-                        (val & pair.second) == pair.second)
+                        (val & pair.second) == pair.second &&

```

Comment:
This again looks like a `WI_IsFlagSet`, which I _think_ also does  `WI_IsSingleFlagSet`